grpc-bridge-runtime/src/main/java/dev/resteasy/grpc/bridge/runtime/servlet/HttpServletRequestImpl.java

```diff
@@ -129,7 +130,7 @@ public HttpServletRequestImpl(final ServletResponse servletResponse, final Servl
         if ("com.google.protobuf.Any".equals(retn)) {
             acceptList = new ArrayList<String>();
             acceptList.add("true");
-            headers.put(GRPC_RETURN_RESPONSE, acceptList);
+            headers.put(ANY, acceptList);
```

Comment:
Is `ANY` a valid HTTP header? For http/2 header keys must be lowercase. 
Per [RFC-7540](https://datatracker.ietf.org/doc/html/rfc7540#section-8.1.2)
> However, header field names MUST be converted to lowercase prior to their encoding in HTTP/2. A request or response containing uppercase header field names MUST be treated as malformed ([Section 8.1.2.6](https://httpwg.org/specs/rfc7540.html#malformed)).

Or is this not an HTTP header?

Reply:
> is this not an HTTP header

Actually, I'm using it to communicate with *MessageBodyReaderWriter. I think I can remove it once *MessageBodyReaderWriter is done with it.

Reply:
Removed